src/components/PressableOpacity.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { ReactElement, ReactNode, useState } from 'react'
+import { StyleProp, ViewStyle } from 'react-native'
+import styled from 'styled-components/native'
+
+interface Props {
+  children: ReactNode
+  onPress: () => void
+  style?: StyleProp<ViewStyle>
```

Comment:
required to apply styles on custom components with styled components f.e. `styled(PressableOpacity)`